apps/api-catalogue/screens/ServiceList/ServiceList.tsx

```diff
@@ -190,6 +182,233 @@ export function ServiceList({ pageContent, filterStrings }: ServiceListProps) {
         </Box>
       }
       left={
+        <FilterSearch
```

Comment:
Maybe the search filter could be a seperate component?

Reply:
FilterSearch is a seperate component.

Reply:
I see, you have already done that, so we will do that

Reply:
> Few comments



> Few comments

I created a story in storybook, there are examples, on how to use the component